src/index.html

```diff
@@ -30,6 +30,7 @@ <h1>Today's To Do</h1><i class="fa fa-refresh"></i>
       <p class="clrCompleted">Clear all completed</p>
     </section>
   </main>
+  <!-- <script type="module" src="index.js"></script> -->
```

Comment:
- [ ] Please kindly remove this introduced commented code.

Reply:
Asked change implemented